test/Air/Air.test.js

```diff
@@ -756,6 +756,50 @@ describe('#AirService', () => {
       });
     });
 
+    it('should set FOID and retry ticketing if PNR busy but not get into infinite cycle on FOID error', function () {
+      this.timeout(2000);
+      const params = { pnr: 'PNR001' };
+
+      const getUniversalRecordByPNR = sinon.spy(
+        () => {
+          console.log('UniversalRecord requested');
+          return Promise.resolve(getURbyPNRSampleTicketed);
+        }
+      );
+      const ticketResponses = [
+        Promise.resolve(),
+        Promise.reject(new AirRuntimeError.TicketingPNRBusy([2])),
+        Promise.reject(new AirRuntimeError.TicketingFoidRequired([1])),
```

Comment:
Test is not correct, because for AirRuntimeError.TicketingFoidRequired response will ALWAYS be `AirRuntimeError.TicketingFoidRequired `